lib/modules/manager/nuget/artifacts.spec.ts

```diff
@@ -18,7 +18,6 @@ jest.mock('./util');
 const { getConfiguredRegistries, getDefaultRegistries } = mocked(util);
 const hostRules = mocked(_hostRules);
 
-// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion
```

Comment:
again? wrong merge?

Reply:
no i just did lint-fix, because it failed lint, apparently the new lint version requires this